pyo3-macros-backend/src/pymethod.rs

```diff
@@ -640,25 +674,31 @@ pub fn impl_py_getter_def(
                 mutable: false,
                 span: Span::call_site(),
             }
-            .receiver(cls, ExtractErrorMode::Raise, &mut holders);
+            .receiver(cls, ExtractErrorMode::Raise, &mut holders, ctx);
             let field_token = if let Some(ident) = &field.ident {
                 // named struct field
                 ident.to_token_stream()
             } else {
                 // tuple struct field
                 syn::Index::from(field_index).to_token_stream()
             };
-            quotes::map_result_into_ptr(quotes::ok_wrap(quote! {
-                ::std::clone::Clone::clone(&(#slf.#field_token))
-            }))
+            quotes::map_result_into_ptr(
+                quotes::ok_wrap(
+                    quote! {
+                        ::std::clone::Clone::clone(&(#slf.#field_token))
+                    },
+                    ctx,
+                ),
+                ctx,
+            )
```

Comment:
I wonder if it makes sense for future refactoring like moving these `quotes` functions to be methods on `Ctx`. A follow up for anyone interested I think, rather than part of this PR.